src/helpers/color.ts

```diff
@@ -367,21 +367,132 @@ const ColorsList = [
   "rgb(158,218,229)",
 ];
 
-export function getNthColor(index: number): Color {
-  return ColorsList[index % ColorsList.length];
+const COLORS_SM = [
+  "#4EA7F2", // Blue
+  "#EA6175", // Red
+  "#43C5B1", // Teal
+  "#F4A261", // Orange
+  "#8481DD", // Purple
+  "#FFD86D", // Yellow
+];
+const COLORS_MD = [
+  "#4EA7F2", // Blue #1
+  "#3188E6", // Blue #2
+  "#43C5B1", // Teal #1
+  "#00A78D", // Teal #2
+  "#EA6175", // Red #1
+  "#CE4257", // Red #2
+  "#F4A261", // Orange #1
+  "#F48935", // Orange #2
+  "#8481DD", // Purple #1
+  "#5752D1", // Purple #2
+  "#FFD86D", // Yellow #1
+  "#FFBC2C", // Yellow #2
+];
+const COLORS_LG = [
+  "#4EA7F2", // Blue #1
+  "#3188E6", // Blue #2
+  "#056BD9", // Blue #3
+  "#A76DBC", // Violet #1
+  "#7F4295", // Violet #2
+  "#6D2387", // Violet #3
+  "#EA6175", // Red #1
+  "#CE4257", // Red #2
+  "#982738", // Red #3
+  "#43C5B1", // Teal #1
+  "#00A78D", // Teal #2
+  "#0E8270", // Teal #3
+  "#F4A261", // Orange #1
+  "#F48935", // Orange #2
+  "#BE5D10", // Orange #3
+  "#8481DD", // Purple #1
+  "#5752D1", // Purple #2
+  "#3A3580", // Purple #3
+  "#A4A8B6", // Gray #1
+  "#7E8290", // Gray #2
+  "#545B70", // Gray #3
+  "#FFD86D", // Yellow #1
+  "#FFBC2C", // Yellow #2
+  "#C08A16", // Yellow #3
+];
+const COLORS_XL = [
+  "#4EA7F2", // Blue #1
+  "#3188E6", // Blue #2
+  "#056BD9", // Blue #3
+  "#155193", // Blue #4
+  "#A76DBC", // Violet #1
+  "#7F4295", // Violet #1
+  "#6D2387", // Violet #1
+  "#4F1565", // Violet #1
+  "#EA6175", // Red #1
+  "#CE4257", // Red #2
+  "#982738", // Red #3
+  "#791B29", // Red #4
+  "#43C5B1", // Teal #1
+  "#00A78D", // Teal #2
+  "#0E8270", // Teal #3
+  "#105F53", // Teal #4
+  "#F4A261", // Orange #1
+  "#F48935", // Orange #2
+  "#BE5D10", // Orange #3
+  "#7D380D", // Orange #4
+  "#8481DD", // Purple #1
+  "#5752D1", // Purple #2
+  "#3A3580", // Purple #3
+  "#26235F", // Purple #4
+  "#A4A8B6", // Grey #1
+  "#7E8290", // Grey #2
+  "#545B70", // Grey #3
+  "#3F4250", // Grey #4
+  "#FFD86D", // Yellow #1
+  "#FFBC2C", // Yellow #2
+  "#C08A16", // Yellow #3
+  "#936A12", // Yellow #4
+];
+
+/**
+ * @param {string} colorScheme
+ * @param {string} paletteName
+ * @returns {array}
+ */
+export function getColors(palette: string) {
+  switch (palette) {
+    case "sm":
+      return COLORS_SM;
+    case "md":
+      return COLORS_MD;
+    case "lg":
+      return COLORS_LG;
+    default:
+      return COLORS_XL;
+  }
+}
+
+export function getNthColor(index: number, palette: string): Color {
+  const colors = getColors(palette);
+  return colors[index % ColorsList.length];
+}
+
+export function getPaletteName(quantity: number) {
```

Comment:
Do you have to go through a palette name ? Is returning the palette directly possible ?